config/locales/models/en.yml

```
@@ -90,6 +90,9 @@ en:
               required: can't be blank
         comment:
           attributes:
+            comment_content:
+              duplicate_comment: You've already left this comment here. (It may not appear right away for performance reasons.)
+              format: "%{message}"
```

This was also my initial thought. But it turns out we can't use this error format here because there are other error messages that need to have "Comment content" at the start. So, this should not override the general format and instead use the `^` at the start of the message itself.
```suggestion
              duplicate_comment: ^You've already left this comment here. (It may not appear right away for performance reasons.)
```